### DISCUSSION TITLE
Daily changing background

### DISCUSSION THREAD

**Discussion by dgibbs64** [linuxserver/Heimdall#992] [Ideas]
I would like the ability to have an "image of the day" for Heimdall similar to bing images.

I would recommend using https://picsum.photos/ as it has a simple url to get the images. Plus all images are royalty-free and provided by https://unsplash.com/

The image could then be cached for 24 hours.

**Comment by martadinata666** [comment]
in simple terms you can do this by using `curl` and `cron` to override default image at `public/img/bg1.jpg` per 24h.
